cmd/neofs-node/config/engine/shard/blobstor/config.go

```diff
@@ -88,6 +88,15 @@ func (x *Config) Compress() bool {
 	)
 }
 
+// UncompressableContentTypes returns value of "compress_skip_content_types" config parameter.
```

Comment:
```suggestion
// UncompressableContentTypes returns value of "disable_compression_content_types" config parameter.
```

Reply:
`compression_exclude_content_types` ? @532910 @realloc

Reply:
exclude is better than disable
`no_compressed_content_types`
`nocompression_for_content_types`
`uncompressed_content_types`
`don't_compress_content_types`

Reply:
`compression_exclude_content_types`?